test_vm/src/lib.rs

```diff
@@ -74,8 +75,11 @@ use std::ops::Add;
 
 pub mod util;
 
-pub struct TestVM<'bs> {
-    pub store: &'bs MemoryBlockstore,
+pub struct TestVM<'bs, BS>
+where
+    BS: Blockstore,
+{
+    pub store: &'bs BS,
     pub state_root: RefCell<Cid>,
```

Comment:
In a follow-up, these should probably both become non-`pub` if they're to be abstracted over by trait.